bedrock/mozorg/templates/mozorg/developer/index.html

```diff
@@ -81,7 +79,7 @@ <h3 class="card-title">{{ _('Firefox DevTools') }}</h3>
 
   <section class="resources">
     <div class="content">
-      <h2 class="section-title"><span>{{ _('Developer Resources') }}</span></h2>
+      <h2 class="section-title"><span>{{ _('Developer resources') }}</span></h2>
```

Comment:
These titles are changing to sentence case. I don't believe the /developer hub page is being localized yet, so this *should* be fine?